modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -221,6 +221,10 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/cryopod, 32)
 			if(mob_occupant.mind)
 				stored_rank = mob_occupant.mind.assigned_role.title
 
+		var/mob/living/carbon/human/human_occupant = occupant
+		if(human_occupant)
+			human_occupant.save_individual_persistence()
```

Comment:
Does this handle the case where someone is added to cryo at the point where they don't have a mind anymore? Say, through admin intervention (which isn't all that uncommon).

Reply:
Fixed